cabal-bundler/tests/Tests.hs

```diff
@@ -42,13 +45,19 @@ main = do
 
             return (toUTF8BS script)
 
-        , golden "fetch-with-curl.sh"$  do
+        , golden "fetch-with-curl.sh"$ do
             planPath <- makeAbsoluteFilePath $ pwd FP.</> "fixtures/cabal-fmt.plan.json"
             plan     <- liftIO $ P.decodePlanJson (toFilePath planPath)
             script   <- generateCurl pn exeName plan meta
 
             return (toUTF8BS script)
 
+        , golden "openbsd-ports.txt" $ do
```

Comment:
Very cool, thanks for adding this test!

Reply:
My pleasure. I couldn't ask you to write the first test, it would have been embarrassing :)